src/application/selector/json/JsonResultFormatter.js

```diff
@@ -0,0 +1,14 @@
+class JsonResultFormatter {
```

Comment:
The responsibility of this class is to convert result of data selecting into single string variable. This string is not necessarily a valid json (it could also be a raw value in string). The input is different for Json class and different for Xml class, so generic `ResultsToJsonConverter` could be an incomprehension. What about `JsonResultConverter`?

Reply:
Sound good.